pages/index.vue

```diff
@@ -13,6 +13,10 @@
       icon: "#i-spotify",
       href: "https://open.spotify.com/playlist/21phkh5dZrZtO4E9Bf9qUy",
     },
+    {
+      icon: "#i-lock",
+      href: "/dashboard",
+    }
```

Comment:
```suggestion
    }
```
Dashboard should not be linked to; viewers will be seeing this page.

Reply:
It's more of a temporary thing because I'm assuming the whole frontpage will be redesigned

Reply:
Eventually, will be updating the site to this very soon tho.